library/pk.c

```diff
@@ -60,6 +60,12 @@ void mbedtls_pk_init(mbedtls_pk_context *ctx)
 {
     ctx->pk_info = NULL;
     ctx->pk_ctx = NULL;
+#if defined(MBEDTLS_ECP_C) && defined(MBEDTLS_USE_PSA_CRYPTO)
+    mbedtls_platform_zeroize(ctx->pk_raw, sizeof(ctx->pk_raw));
```

Comment:
Nit: Actually `memset(0)` would fine here, `platform_zeroize()` is mostly useful in `_free()` functions.